contribs/gmf/src/directives/drawfeature.js

```diff
@@ -157,7 +168,20 @@ gmf.DrawfeatureController = function($scope, $timeout, ngeoDecorateInteraction,
    * @private
    */
   this.modify_ = new ngeo.interaction.Modify({
-    features: this.selectedFeatures_
+    features: this.selectedFeatures_,
+    style: new ol.style.Style({
+      image: new ol.style.RegularShape({
+        radius: 6,
+        points: 4,
+        angle: Math.PI / 4,
+        fill: new ol.style.Fill({
+          color: [255, 255, 255, 0.5]
+        }),
+        stroke: new ol.style.Stroke({
+          color: [0, 0, 0, 1]
+        })
+      })
+    })
```

Comment:
The style used here is the same as the style used for vertices. This should be factorized in my opinion.